src/cascadia/TerminalApp/ColorScheme.cpp

```diff
@@ -1,222 +1,237 @@
-    {
-        const auto nameFromJson = GetWstringFromJson(name);
-        return nameFromJson == _schemeName;
-    }
-    return false;
-}
-
-// Method Description:
-// - Layer values from the given json object on top of the existing properties
-//   of this object. For any keys we're expecting to be able to parse in the
-//   given object, we'll parse them and replace our settings with values from
-//   the new json object. Properties that _aren't_ in the json object will _not_
-//   be replaced.
-// Arguments:
-// - json: an object which should be a partial serialization of a ColorScheme object.
-// Return Value:
-// <none>
-void ColorScheme::LayerJson(const Json::Value& json)
-{
-    if (auto name{ json[JsonKey(NameKey)] })
-    {
-        _schemeName = winrt::to_hstring(name.asString());
-    }
-    if (auto fgString{ json[JsonKey(ForegroundKey)] })
-    {
-        const auto color = Utils::ColorFromHexString(fgString.asString());
-        _defaultForeground = color;
-    }
-    if (auto bgString{ json[JsonKey(BackgroundKey)] })
-    {
-        const auto color = Utils::ColorFromHexString(bgString.asString());
-        _defaultBackground = color;
-    }
-
-    // Legacy Deserialization. Leave in place to allow forward compatibility
-    if (auto table{ json[JsonKey(TableKey)] })
-    {
-        int i = 0;
-
-        for (const auto& tableEntry : table)
-        {
-            if (tableEntry.isString())
-            {
-                auto color = Utils::ColorFromHexString(tableEntry.asString());
-                _table.at(i) = color;
-            }
-            i++;
-        }
-    }
-
-    int i = 0;
-    for (const auto& current : TableColors)
-    {
-        if (auto str{ json[JsonKey(current)] })
-        {
-            const auto color = Utils::ColorFromHexString(str.asString());
-            _table.at(i) = color;
-        }
-        i++;
-    }
-}
-
-std::wstring_view ColorScheme::GetName() const noexcept
-{
-    return { _schemeName };
-}
-
-std::array<COLORREF, COLOR_TABLE_SIZE>& ColorScheme::GetTable() noexcept
-{
-    return _table;
-}
-
-COLORREF ColorScheme::GetForeground() const noexcept
-{
-    return _defaultForeground;
-}
-
-COLORREF ColorScheme::GetBackground() const noexcept
-{
-    return _defaultBackground;
-}
-
-// Method Description:
-// - Parse the name from the JSON representation of a ColorScheme.
-// Arguments:
-// - json: an object which should be a serialization of a ColorScheme object.
-// Return Value:
-// - the name of the color scheme represented by `json` as a std::wstring optional
-//   i.e. the value of the `name` property.
-// - returns std::nullopt if `json` doesn't have the `name` property
-std::optional<std::wstring> TerminalApp::ColorScheme::GetNameFromJson(const Json::Value& json)
-{
-    if (const auto name{ json[JsonKey(NameKey)] })
-    {
-        return GetWstringFromJson(name);
-    }
-    return std::nullopt;
-}
+// Copyright (c) Microsoft Corporation.
```

Comment:
Why does it think this entire file changed?

Reply:
some weird whitespace only change 🤷‍♂, i reverted it though.